Danki2/Assets/Scripts/Actor/Actor.cs

```diff
@@ -47,4 +47,17 @@ public void AddEffect(Effect effect)
     {
         _effectTracker.AddEffect(effect);
     }
+
+    protected void MoveAlongVector(Vector3 vec)
+    {
+        vec.Normalize();
+        vec *= _statsManager[Stat.Speed];
+        transform.Translate(vec);
```

Comment:
This is getting called every frame by the player if if the vector is (0, 0, 0). This could get a bit expensive I reckon, maybe we should have a check at the start of this method for a zero vector? Or not call this method from the player if there's no input? Or maybe both?